testing/felcat_config.yaml

```diff
@@ -0,0 +1,25 @@
+servers:
+  - server_name: Caltech
+    base_url: https://felcat.caltech.edu/rnaget/
+    implemented:
+      projects: true
+      studies: true
+      expressions: true
+      continuous: false
+    
+    projects:
+      - id: 43378a5d48364f9d8cf3c3d5104df560
+        filters:
+          version: 1.0
+          name: PCAWG
+    
+    studies:
+      - id: 6cccbbd76b9c4837bd7342dd616d0fec
+        filters:
+          version: 1.0
+          name: PCAWG
+    
+    expressions:
+      - id: 2a7ab5533ef941eaa59edbfe887b58c4
+        filters:
+          studyID: 6cccbbd76b9c4837bd7342dd616d0fec
```

Comment:
format is now a required filter for each expression, as format must be specified in the /expressions/search endpoint. The expression object definition should look like this:

```
expressions:
  - id: 2a7ab5533ef941eaa59edbfe887b58c4
    filters:
    studyID: 6cccbbd76b9c4837bd7342dd616d0fec
    format: loom
```